cluster-controller/src/main/java/io/strimzi/controller/cluster/resources/KafkaCluster.java

```diff
@@ -112,31 +112,31 @@ public static String headlessName(String cluster) {
     /**
      * Create a Kafka cluster from the related ConfigMap resource
      *
-     * @param cm    ConfigMap with cluster configuration
-     * @return   Kafka cluster instance
+     * @param kafkaClusterCm ConfigMap with cluster configuration
```

Comment:
Why did you renamed it? To make our review harder? :-)

Reply:
@scholzj see my earlier review comment about the fact that we have three "clusters" each sharing a single cm, rather than having their own.